test/e2e/lib/gutenberg/blocks/layout-grid-block-component.js

```diff
@@ -78,8 +78,11 @@ class LayoutGridBlockComponent extends GutenbergBlockComponent {
 			By.css( `button.editor-block-list-item-${ blockClass.blockName.replace( '/', '-' ) }` )
 		);
 
+		// @TODO Remove the `deprecatedInsertedBlockLocator` once we get 10.5.0 in production.
```

Comment:
Had to revert this commit, it looks like some blocks are still using this "deprecated" locator.